pkg/generators/forwarding/fluentd/output_fluentd_buffer.go

```diff
@@ -0,0 +1,122 @@
+package fluentd
+
+import (
+	"fmt"
+
+	loggingv1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+)
+
+const (
+	// Buffer size defaults
+	defaultOverflowAction = "block"
+
+	// Flush buffer defaults
+	defaultFlushThreadCount = "2"
+	defaultFlushMode        = "interval"
+	defaultFlushInterval    = "1s"
```

Comment:
In the interest of being safe, I recommend reusing `TimeUnit` which has some attached conversion methods (or it did) and converting all of these to seconds without unit

Reply:
As mentioned below, `TimeUnit` is related to elasticsearch and fluentd has its on limitation on the time config parameter type. We are safe from an API standpoint, because the validation is using the same pattern as fluentd's parser. Any wrong input will never make it into our API stored ClusterLogging CRs.